src/main/kotlin/com/dansplugins/factionsystem/claim/MfClaimService.kt

```diff
@@ -60,6 +60,24 @@ class MfClaimService(private val plugin: MedievalFactions, private val repositor
         return false
     }
 
+    // Checks whether a set of chunks has at least one chunk that is adjacent to an existing claim. Works across multiple worlds.
+    fun claimIsAdjacent(chunks: Set<Chunk>, id: MfFactionId): Boolean
+    {
```

Comment:
Small note: The project uses a particular brace format, whereas opening braces appear on the same line as statements. It's not a big deal since we want to start using an auto-formatter anyway.